### PR TITLE
Spelling: Checks.py error messages reworked

### PR DESCRIPTION
**Pull request by comradekingu**
However cute this is,
[The Celery does not process tasks](https://github.com/WeblateOrg/weblate/blob/159c4574b74b7d61feb29344211b352f097d66fb/weblate/utils/checks.py#L200)
 [The Celery is not configured](https://github.com/WeblateOrg/weblate/blob/159c4574b74b7d61feb29344211b352f097d66fb/weblate/utils/checks.py#L208)

Celery queue, worker?

### CODE REVIEW

File: weblate/utils/checks.py [hunk 1]
```diff
@@ -127,8 +127,8 @@ def is_celery_queue_long():
     """
     Checks whether celery queue is too long.
 
-    It does trigger if it is too long for at least one hour. This way we filter out
-    peaks and avoid flipping warning on big operations (eg. site-wide autotranslate).
+    It does trigger if it is too long for at least one hour. This way peaks are
+    filtered out, and no warning need be issued for big operations (e.g. site-wide autotranslate).
```
Comment (comradekingu) [inline weblate/utils/checks.py:131]:
```suggestion
    filtered out, and no warning need be issued for big operations (e.g. site-wide autotranslation).
```

File: weblate/utils/checks.py [hunk 2]
```diff
@@ -216,7 +216,7 @@ def check_celery(app_configs, **kwargs):
         errors.append(
             weblate_check(
                 "weblate.C030",
-                "The Celery beats scheduler is not executing periodic tasks "
+                "The Celery beat scheduler is not executing periodic tasks "
```
Comment (comradekingu) [inline weblate/utils/checks.py:219]: As per https://docs.celeryproject.org/en/stable/reference/celery.beat.html

File: weblate/utils/checks.py [hunk 3]
```diff
@@ -270,27 +270,27 @@ def check_settings(app_configs, **kwargs):
     if not settings.ADMINS or "noreply@weblate.org" in (x[1] for x in settings.ADMINS):
         errors.append(
             weblate_check(
-                "weblate.E011", "The site admins seem to be wrongly configured", Error
+                "weblate.E011", "E-mail addresses for site admins is misconfigured", Error
```
Comment (comradekingu) [inline weblate/utils/checks.py:273]: Or usernames?

File: weblate/utils/checks.py [hunk 4]
```diff
@@ -270,27 +270,27 @@ def check_settings(app_configs, **kwargs):
...
 
     if settings.SERVER_EMAIL in DEFAULT_MAILS:
         errors.append(
-            weblate_check("weblate.E012", "The server email has default value")
+            weblate_check("weblate.E012", "The server e-mail address should be changed from its default value")
         )
     if settings.DEFAULT_FROM_EMAIL in DEFAULT_MAILS:
         errors.append(
-            weblate_check("weblate.E013", "The default from email has default value")
+            weblate_check("weblate.E013", "The default from e-mail should be changed from its default value")
```
Comment (comradekingu) [inline weblate/utils/checks.py:283]:
```suggestion
            weblate_check("weblate.E013", "The default \"From\" e-mail address should be changed from its default value")
```

File: weblate/utils/checks.py [hunk 5]
```diff
@@ -354,15 +354,15 @@ def check_site(app_configs, **kwargs):
...
     return errors
 
 
 def check_perms(app_configs=None, **kwargs):
-    """Check we can write to data dir."""
+    """Check that the data dir can be written to."""
     errors = []
     uid = os.getuid()
-    message = "Path {} is owned by different user, check your DATA_DIR settings."
+    message = "The path {} is owned by different user, check your DATA_DIR settings."
```
Comment (comradekingu) [inline weblate/utils/checks.py:365]: Should {} be quoted?

File: weblate/utils/checks.py (hunk 3 above)
Comment (nijel) [reply weblate/utils/checks.py:273]: It's about this: https://docs.weblate.org/en/latest/admin/install.html#production-admins

So e-mail seems good fit here.

File: weblate/utils/checks.py (hunk 5 above)
Comment (nijel) [reply weblate/utils/checks.py:365]: It's file system path, probably not necesary to quote it...

**Comment by nijel** [comment]
> Celery queue, worker?

We're checking workers, the queues are hard coded in our configuration.

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Pull request merged**